### ISSUE TITLE
Check If Number Has Whatsapp?

### ISSUE THREAD

**Issue by Sharaf-Mansour** [orkestral/venom#1903]
I looked at previous issues and also read the docs but could not find a section about a function to check if the number provided has WhatsApp or not.
It should be something like ```hasWhats('00000000000')``` that returns true if it has and false if not.

**Comment by renandecarlo** [comment]
There is `checkNumberStatus`

**Comment by Sharaf-Mansour** [comment]
It requires you to be logged in @renandecarlo

**Comment by renandecarlo** [comment]
Every function of the bot requires you to be logged in

**Comment by Sharaf-Mansour** [comment]
Yes, is there a way to track the API back to the original endpoint to make a phone checker that does not require a login ?
